packages/app/src/types/wallet/index.ts

```diff
@@ -7,3 +7,8 @@ export interface ICheckMnemonicArgs {
   mnemonic: string;
   strict?: boolean;
 }
+
+export interface IChangeMnemonicPassword {
```

Comment:
I think it doesn't make sense to have to same types but with different names, could we unify them with one generic name? 
- `IResetPasswordArgs`, `IChangeMnemonicPassword`